_posts/en/newsletters/2020-10-07-newsletter.md

```diff
@@ -0,0 +1,198 @@
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta][lnd 0.11.1-beta] is the release for a new minor
+  version.  Its release notes summarize the changes as "a number of
+  reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor
+  commitments][topic anchor outputs] spec compliant."
+
+- [HWI 1.2.0-rc.1][HWI 1.2.0] is a release candidate that adds support
+  for a new hardware device and contains multiple bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19898][] log: print unexpected version warning in validation log category FIXME:jonatack
+
+- [Bitcoin Core #15367][] feature: Added ability for users to add a startup command FIXME:dongcarl
+
+- [Bitcoin Core #19723][] ignores unknown P2P protocol messages sent
+  before the `verack` (version acknowledgment) message.  This allows a
+  node to send its `version` message, followed by zero or more special
+  messages identifying any special features it supports, and then its
+  `verack` message indicating it has received the remote node's
+  `version` message and is ready to begin normal relay. See [Newsletter
+  #111][news111 pre-verack] for previous discussion about this proposed
+  method for protocol negotiation.
+
+- [Bitcoin Core #19725][] updates the `getpeerinfo` RPC.  Its results
+  now return a new `connection_type` field that indicates the
+  reason the node either opened that connection to an outbound peer or
+  accepted in inbound connection from the peer.  The existing `addnode`
+  field in the RPC output is deprecated (not returned by default);
+  connections manually requested by the user now show up as
+  `connection_type: manual`.
+
+- [Bitcoin Core #18309][] allows each ZeroMQ configuration parameter to
+  be specified multiple times.  If the same parameter is specified more
+  than once, each listed IP address and port will receive notifications.
+  Previously, only the first provided address/port would receive a
+  notification.
+
+- [Bitcoin Core #19501][] updates the `sendtoaddress` and `sendmany`
+  RPCs with a new optional `verbose` parameter that will return what
+  mechanism was used to select the transaction's feerate, such as
+  whether the user manually selected the feerate, an appropriate feerate
+  was automatically selected, or the configured fallback feerate was used.
+
+- [Bitcoin Core #20003][] will result in the program immediately exiting
```

Comment:
s/the program/bitcoind

Reply:
I say "the program" here because you can run Bitcoin Core under several names (`bitcoind`, `bitcoin-qt`, `bitcoin-node` (with certain --configure options), `bitcoin-core` (in the Snap), etc...).  No matter which name you use, the `--proxy` behavior described here should apply.